people.go

```diff
@@ -328,22 +333,25 @@ func (p People) ForEach(f func(int64, *Person) bool) {
 
 // FindPeople returns all the people in the database or from the disk cache.
 func FindPeople(ctx context.Context, connString string, cachePath string, blacklist Blacklist) (
-	People, map[string]int, error) {
+	People, map[string]int, map[string]int, error) {
 	persons, err := findRawPersons(ctx, connString, cachePath)
 	reporter.Commit("people found", len(persons))
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 	people, err := newPeople(persons, blacklist)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 	nameFreqs, err := getNamesFreqs(persons)
 	if err != nil {
-		return people, nil, err
+		return people, nil, nil, err
 	}
-
-	return people, nameFreqs, nil
+	emailFreqs, err := getEmailsFreqs(persons)
+	if err != nil {
+		return people, nameFreqs, nil, err
+	}
+	return people, nameFreqs, emailFreqs, nil
```

Comment:
```suggestion
	return people, nameFreqs, emailFreqs, err
```

Because if `err` is not `nil`, `emailFreqs` will be `nil`, and vice versa. Thus you can remove the upper 3 lines.